js/mocha-tests/.babelrc

```diff
@@ -0,0 +1,11 @@
+{
+  "presets": [
+    "es2015",
+    "stage-0"
```

Comment:
do you mind removing `stage-0` here? I feel it makes sense to have a stable project that depends on an agreed upon language specification :)